docs/introduction/tasks.md

```diff
@@ -1,148 +1,238 @@
 # Tasks
 
-The **Inductiva API** client provides the ability to run complex simulations and
-contains easy-to-use tools to manage them. Each simulation was submitted - with
-`simulator.run()` - runs asynchronously and returns a `Task` object that contains
-methods to manage the simulation.
+At the heart of the API lies the concept of a `Task`.
```

Comment:
I think we are using this image ("at the heart") elsewhere. I think it is fine now, but then Maya can help with cleaning up this sort of repeated pattarns.